--- mariadb.cnf.orig	2017-05-14 23:13:15.000000000 +0000
+++ mariadb.cnf	2017-07-08 11:03:29.919584548 +0000
@@ -3,7 +3,7 @@
 
 [client]
 # Default is Latin1, if you need UTF-8 set this (also in server section)
-#default-character-set = utf8 
+default-character-set = utf8 
 
 [mysqld]
 #
@@ -11,9 +11,9 @@
 # 
 # Default is Latin1, if you need UTF-8 set all this (also in client section)
 #
-#character-set-server  = utf8 
-#collation-server      = utf8_general_ci 
-#character_set_server   = utf8 
-#collation_server       = utf8_general_ci 
+character-set-server  = utf8 
+collation-server      = utf8_general_ci 
+character_set_server   = utf8 
+collation_server       = utf8_general_ci 
 # Import all .cnf files from configuration directory
 !includedir /etc/mysql/mariadb.conf.d/
