--- 000-default.conf.orig	2016-03-19 09:48:35.000000000 +0000
+++ 000-default.conf	2018-01-16 09:59:59.875227151 +0000
@@ -26,6 +26,10 @@
 	# following line enables the CGI configuration for this host only
 	# after it has been globally disabled with "a2disconf".
 	#Include conf-available/serve-cgi-bin.conf
+
+	# Redirect all requests to https.
+	RewriteEngine on
+	RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,R=permanent]
 </VirtualHost>
 
 # vim: syntax=apache ts=4 sw=4 sts=4 sr noet
