--- my.cnf.orig	2017-05-14 23:13:15.000000000 +0000
+++ my.cnf	2017-07-08 11:11:26.296831240 +0000
@@ -133,6 +133,8 @@
 default_storage_engine	= InnoDB
 # you can't just change log file size, requires special procedure
 #innodb_log_file_size	= 50M
+innodb_large_prefix     = 1
+innodb_file_format      = barracuda
 innodb_buffer_pool_size	= 256M
 innodb_log_buffer_size	= 8M
 innodb_file_per_table	= 1
@@ -159,7 +161,7 @@
 #wsrep_on=ON
 #wsrep_provider=
 #wsrep_cluster_address=
-#binlog_format=row
+binlog_format=mixed
 #default_storage_engine=InnoDB
 #innodb_autoinc_lock_mode=2
 #
