--- mariadb.cnf.orig	2017-01-17 22:20:01.000000000 +0000
+++ mariadb.cnf	2016-12-06 16:39:48.000000000 +0000
@@ -3,7 +3,7 @@
 
 [client]
 # Default is Latin1, if you need UTF-8 set this (also in server section)
-#default-character-set = utf8 
+default-character-set = utf8 
 
 [mysqld]
 #
@@ -11,7 +11,7 @@
 # 
 # Default is Latin1, if you need UTF-8 set all this (also in client section)
 #
-#character-set-server  = utf8 
-#collation-server      = utf8_general_ci 
-#character_set_server   = utf8 
-#collation_server       = utf8_general_ci 
+character-set-server  = utf8 
+collation-server      = utf8_general_ci 
+character_set_server   = utf8 
+collation_server       = utf8_general_ci 
