--- apache24.conf.orig	2017-02-15 11:08:04.738220210 +0000
+++ apache24.conf	2017-02-15 11:10:17.595192881 +0000
@@ -1,12 +1,14 @@
 Alias /munin /var/cache/munin/www
 <Directory /var/cache/munin/www>
         Require local
+        Include /etc/apache2/misc/my-current-admin-host.conf
         Options None
 </Directory>
 
 ScriptAlias /munin-cgi/munin-cgi-graph /usr/lib/munin/cgi/munin-cgi-graph
 <Location /munin-cgi/munin-cgi-graph>
         Require local
+        Include /etc/apache2/misc/my-current-admin-host.conf
 	<IfModule mod_fcgid.c>
 	    SetHandler fcgid-script
 	</IfModule>
